src/partials/about.html

```diff
@@ -5,7 +5,7 @@ <h2 class="about__header">how it&#8217;s made&#63;</h2>
     <div class="about__description-space">
       <picture>
         <source srcset="/src/images/about/desktop/about_cow_desktop.png 1x, /src/images/about/desktop/about_cow_desktop@2x.png 2x"
-                media="(min-width: 1200px)">
+                media="(min-width: 1280px)">
         <source srcset="/src/images/about/tablet/about_cow_tablet.png 1x, /src/images/about/tablet/about_cow_tablet@2x.png 2x"
                 media="(min-width: 768px)">
         <source srcset="/src/images/about/about_cow.png 1x, /src/images/about/about_cow@2x.png 2x"
```

Comment:
1. Dla <div class="about__description"> zmienił bym nazwę klasy na about__descriptions
2. Dla <p class="about__description-item-first"> zmienił bym nazwę na about__description-upper
3. Dla <p class="about__description-item-second"> zmienił bym nazwę na about__description-lower
4. Dla <button class="aboute__description-button"> zmienił bym nazwę na about__btn , ponieważ nie ma żadnego innego przycisku w tej sekcji to nie ma potrzeby wydłużać klasę o człon "description"